config/schemas/6/rapidast_schema.json

```diff
@@ -0,0 +1,605 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "RapiDAST Configuration",
+    "description": "Schema for validating RapiDAST configuration files",
+    "type": "object",
+    "properties": {
+        "config": {
+            "type": "object",
+            "properties": {
+                "configVersion": {
+                    "default": 0,
+                    "$comment": "If no version is specified, the config converter will default to version 0",
+                    "type": "integer"
+                },
+                "base_results_dir": {
+                    "default": "./results",
+                    "type": "string"
+                },
+                "tls_verify_for_rapidast_downloads": {
+                    "default": true,
+                    "type": "boolean"
+                },
+                "environ": {
+                    "type": "object",
+                    "properties": {
+                        "envFile": {
+                            "type": "string"
+                        }
+                    },
+                    "required": []
```

Comment:
The code [supports](https://github.com/RedHatProductSecurity/rapidast/blob/validate-config-jsonschema/rapidast.py#L37) an empty environ block.

Reply:
yes

Reply:
Fixed